core/src/main/java/io/grpc/Context.java

```diff
@@ -49,13 +46,11 @@
 import javax.annotation.Nullable;
 
 /**
- * A context propagation mechanism which carries deadlines, cancellation signals,
- * and other scoped values across API boundaries and between threads. Examples of functionality
- * propagated via context include:
+ * A context propagation mechanism which can carry scoped-values across API boundaries and between
+ * threads. Examples of state propagated via context include:
  * <ul>
- *   <li>Deadlines for a local operation or remote call.</li>
  *   <li>Security principals and credentials.</li>
- *   <li>Local and distributed tracing context.</li>
+ *   <li>Local and distributed tracing information.</li>
```

Comment:
I'd opened an issue recently about these comments, and my understanding was that tracing information will not propagate "across API boundaries" but that the data which crosses API boundaries is limited to deadlines and cancellations.  Instead, if a user wanted to propagate tracing information(or arbitrary metadata), they would need to write a custom interceptor to read the context and add these values in a header.  I still think this can be made more clear here how users are expected to propagate arbitrary metadata outside of cancellation and deadlines.